build-android-arm64-v8a.sh

```
@@ -68,9 +68,14 @@ fi
 
 echo "ANDROID_NDK: $ANDROID_NDK"
 sleep 1
-onnxruntime_version=1.24.3
-
-if [ $BUILD_SHARED_LIBS == ON ]; then
+onnxruntime_version=${SHERPA_ONNX_ONNXRUNTIME_VERSION:-1.24.3}
+
+if [ -n "${SHERPA_ONNXRUNTIME_LIB_DIR:-}" ] && [ -n "${SHERPA_ONNXRUNTIME_INCLUDE_DIR:-}" ]; then
+  echo "Using externally provided ONNX Runtime"
+elif [ -n "${SHERPA_ONNX_ONNXRUNTIME_ROOT:-}" ] && [ $BUILD_SHARED_LIBS == ON ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)